nephio-ansible-install/README.md

```diff
@@ -0,0 +1,154 @@
+
+```yaml
+all:
+  vars:
+    cloud_user: <username that is used to access the VM>
+    github_username: <GitHub username>
+    github_token: <GitHub personal access token>
+    github_organization: <optional, if a GitHub organization is used this should be filled out with the GitHub organization>
+    gitea_username: <gitea username>
+    gitea_password: <gitea password>
+    proxy:
+      http_proxy: 
+      https_proxy:
+      no_proxy:
+    host_os: "linux"  # use "darwin" for MacOS X, "windows" for Windows
+    host_arch: "amd64"  # other possible values: "386","arm64","arm","ppc64le","s390x"
+    tmp_directory: "/tmp"
+    bin_directory: "/usr/local/bin"
+    kubectl_version: "1.25.0"
+    kubectl_checksum_binary: "sha512:fac91d79079672954b9ae9f80b9845fbf373e1c4d3663a84cc1538f89bf70cb85faee1bcd01b6263449f4a2995e7117e1c85ed8e5f137732650e8635b4ecee09"
+    kind_version: "0.17.0"
+    cni_version: "0.8.6"
+    kpt_version: "1.0.0-beta.23"
+    multus_cni_version: "3.9.2"
+    nephio:
+      install_dir: nephio-install
+      packages_url: https://github.com/nephio-project/nephio-packages.git
+    clusters:
+      mgmt: {mgmt_subnet: 172.88.0.0/16, pod_subnet: 10.196.0.0/16, svc_subnet: 10.96.0.0/16}
+      edge1: {mgmt_subnet: 172.89.0.0/16, pod_subnet: 10.197.0.0/16, svc_subnet: 10.97.0.0/16}
+      edge2: {mgmt_subnet: 172.90.0.0/16, pod_subnet: 10.198.0.0/16, svc_subnet: 10.98.0.0/16}
+      region1: {mgmt_subnet: 172.91.0.0/16, pod_subnet: 10.199.0.0/16, svc_subnet: 10.99.0.0/16}
+  children:
+    vm:
+      hosts:
+        <ip address of the VM>:
+```
+
+Some customizations are required to tailor the installation to your environment. Edit the inventory/nephio.yaml file where you update:
+
+- cloud_user: the username that is created to access the VM using SSH
+- github_username: your GitHub username
+- github_token: GitHub access token to access GitHub [GitHub personal access token](https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+- github_organization: (optional) if you use a GitHub organization for the repositories you should add your GitHub organization here, otherwise it uses the GitHub username
+- gitea_username: your [gitea](https://gitea.io) (local repository) username
+- gitea_password: your [gitea](https://gitea.io) (local repository) password
+
+Note: You can choose between using remote GitHub repositories or local gitea repositories for your Nephio environment by setting either the GitHub variables or the gitea variables.
+
+To start running ansible playbooks an ansible environment is required. Below is an example how to install ansible using a virtual environment. The repository scripts rely on the ansible galaxy community collection
+
+```python
+python3 -m venv .venv
+source .venv/bin/activate
+pip install --upgrade pip
+pip install ansible
+pip install pygithub
+ansible-galaxy collection install community.general
```

Comment:
We may also to call out that `ansible-galaxy collection install community.docker` needs to be executed as well. This would be needed wherever the docker_* module is being used

Reply:
fixed https://github.com/nephio-project/one-summit-22-workshop/pull/106/commits/92affdf9ed2c2a920e7843266952f4bd89edc5f3